### PR TITLE
refactor: setup test database with migrations

### PR DESCRIPTION
**Pull request by spiridonovfed**

### CODE REVIEW

File: scheduler/tests/conftest.py [hunk 1]
```diff
@@ -1,49 +1,54 @@
...
 @pytest.fixture
-def testing_session(testing_engine):
+def testing_session(use_temp_env_var, testing_engine):
     sqlalchemy_utils.create_database(testing_engine.url)
-    models.Base.metadata.create_all(bind=testing_engine)
+
+    try:
+        command.upgrade(alembic_cfg, "head")
+    except SQLAlchemyError as e:
+        raise SQLAlchemyError(f"Got an Exception during migrations - {e}")
```
Comment (cakeinsauce) [inline scheduler/tests/conftest.py:50]: Why should we raise the same exception with slightly adjusted message and lost traceback?

Comment (spiridonovfed) [reply scheduler/tests/conftest.py:50]: By doing this I give an understanding, that this Exception is raised specifically during migrations in explicit way.
It helped me a lot during development

Comment (cakeinsauce) [reply scheduler/tests/conftest.py:50]:
```suggestion
    except SQLAlchemyError as e:
        raise SQLAlchemyError("Exception during migrations") from e
```

Comment (spiridonovfed) [reply scheduler/tests/conftest.py]: Agreed - applied your suggestion

**Review by cakeinsauce — APPROVED** [review]

**Pull request merged**